main.go

```diff
@@ -60,6 +61,10 @@ func main() {
 		log.Print("-start only allowed in dev mode")
 		return
 	}
+	if !*devFlag && *headlessFlag {
+		log.Print("-headless only allowed in dev mode")
+		return
```

Comment:
I think we eventually want to be able to do headless even in non-dev mode (we simply would not throw away their session dir I suppose), but that's ok for now.
I see no reason why we have not used log.Fatal[f] instead of log.Print+return. Can you please change all of them in main() to log.Fatal instead?

Reply:
I replaced all the log.Print+return with log.Fatal.